storage/leveldb/leveldbSerial.go

```diff
@@ -258,6 +259,26 @@ func (s *SerialDB) Destroy() error {
 	return err
 }
 
+// DestroyClosed removes the already closed storage medium stored data
+func (s *SerialDB) DestroyClosed() error {
+	s.mutBatch.Lock()
```

Comment:
I think here as well you just need to remove all, as the close should have already been done.

Reply:
done